projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -900,12 +900,18 @@
     }
 
     %igx-grid__td--edited {
-        font-style: italic;
-        color: --var($theme, 'cell-edited-value-color');
+        %grid-cell-text {
+            font-style: italic;
+            color: --var($theme, 'cell-edited-value-color');
+        }
     }
 
     %igx-grid__tr--deleted {
-        text-decoration-line: line-through;
+        %grid-cell-text {
+            font-style: italic;
+            color: igx-color(map-get($theme, 'palette'), 'error');
+            text-decoration-line: line-through;
```

Comment:
@simeonoff Can't this be just `text-decoration` so it'd work in IE/Edge too?
@StefanIvanov Is red part of the design?

Reply:
@damyanpetev text-decoration-line is just a property of the shorthand text-decoration. The `line-through` text decoration is not supported in IE and Edge. As to the color being red, I've discussed it wit @StefanIvanov before implementing the change.

Reply:
@simeonoff Despite being the shorthand, `text-decoration: line-through;` is in the CSS1 spec and [works just fine even in older IE-s](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration#Browser_compatibility). So unless we need any of the decorations, can we use the shorthand for consistency?